src/modules/pos/validator/components/StakeForm/StakeForm.js

```diff
@@ -146,7 +146,7 @@ const StakeForm = ({ t, stakes, account, isStakingTxPending, nextStep, history,
 
   const feedback = validateStakes(
     stakes,
-    BigInt(posToken?.availableBalance),
+    BigInt(posToken?.availableBalance ?? 0),
```

Comment:
This is incorrect? the balance should be based on current account

Reply:
@ManuGowda `posToken` has the balance of the current account in the POS token. It's passed in from the parent component

Reply:
hmm ok, the naming is confusing for sure, thanks for clarification